packages/data/lib/di/data_injector.dart

```diff
@@ -1,23 +1,46 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:data/const.dart';
+import 'package:data/repositories/auth_repository_impl.dart';
+import 'package:data/repositories/credentials_repository_impl.dart';
+import 'package:data/repositories/firestore_repository.dart';
 import 'package:data/repositories/tmdb_images_repository_impl.dart';
 import 'package:data/repositories/trakt_movies_repository_impl.dart';
+import 'package:data/services/analytics_service_impl.dart';
 import 'package:data/services/api_base_service.dart';
 import 'package:data/services/app_config_service_impl.dart';
+import 'package:data/services/facebook_auth_service.dart';
+import 'package:data/services/google_auth_service.dart';
 import 'package:dio/dio.dart';
+import 'package:domain/repositories/auth_repository.dart';
+import 'package:domain/repositories/credentials_repository.dart';
 import 'package:domain/repositories/images_repository.dart';
 import 'package:domain/repositories/movies_repository.dart';
+import 'package:domain/repositories/remote_store_repository.dart';
+import 'package:domain/services/analytics_service.dart';
 import 'package:domain/services/app_config_service.dart';
+import 'package:domain/services/auth_service.dart';
+import 'package:firebase_analytics/firebase_analytics.dart';
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter_facebook_auth/flutter_facebook_auth.dart';
 import 'package:get_it/get_it.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+import 'package:shared_preferences/shared_preferences.dart';
 
 Future<void> initDataInjector(
   String traktApiBaseUrl,
   String traktApiKeyConfigKey,
 ) async {
+  await Firebase.initializeApp();
```

Comment:
do it in the root of the App I think

Reply:
I need Firebase only in data package.

Reply:
agree. but this logic doesn't belong to di, where u put it

Reply:
Fixed.